src/store/allComments.js

```diff
@@ -0,0 +1,76 @@
+
+
+export const allCommentsModule = {
+    state: () => ({
+        allCommentsList: [],
+    }),
+    mutations: {
+        addComment: (state, newCommentInfo) => {
+            const { id, newComment } = newCommentInfo;
+            const addAnswerToComment = (data, parentId, newObject) => {
+                if (!parentId) {
+                    return data.concat(newObject);
+                }
+                for (let i = 0; i < data.length; i++) {
+                    const item = data[i];
+                    if (item.postId === parentId) {
+                        item.answers = item.answers.concat(newObject);
```

Comment:
it's ok to mutate objects in vue as opposed to react. you can read more about it in [docs](https://vuejs.org/guide/essentials/reactivity-fundamentals.html)
```suggestion
                        item.answers.push(newObject);
```